internal/archiver/archiver.go

```diff
@@ -628,42 +628,7 @@ func (arch *Archiver) SaveTree(ctx context.Context, snPath string, atree *Tree,
 	return tree, nil
 }
 
-type fileInfoSlice []os.FileInfo
-
-func (fi fileInfoSlice) Len() int {
-	return len(fi)
-}
-
-func (fi fileInfoSlice) Swap(i, j int) {
-	fi[i], fi[j] = fi[j], fi[i]
-}
-
-func (fi fileInfoSlice) Less(i, j int) bool {
-	return fi[i].Name() < fi[j].Name()
-}
-
-func readdir(filesystem fs.FS, dir string) ([]os.FileInfo, error) {
```

Comment:
I've always been confused by these two functions. It's good to have this cleaned up